.github/workflows/macos.yaml

```diff
@@ -17,14 +17,26 @@ jobs:
         fetch-depth: "0"
     # Get tags not fetched by the checkout action, needed for auto-versioning.
     - run: "git fetch origin +refs/tags/*:refs/tags/*"
+
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
+
+    - name: Use pip cache
+      uses: actions/cache@v2
+      if: startsWith(runner.os, 'macOS')
```

Comment:
Since this workflow is macOS only, is this redundant?

Reply:
Good idea, thank you!  Updated in b4ae2d7.